roles/rook/defaults/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+##########################
+# operator
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+##########################
+# rook
+
+container_registry_rook: quay.io
+rook_image: "{{ container_registry_rook }}/ceph/ceph"
+# renovate: datasource=docker depName=index.docker.io/rook/ceph
+rook_image_tag: "v17.2"
+
+## work directory inside the osism-ansible container:
+rook_work_directory: /tmp/rook/configuration
```

Comment:
Does it makes sense to have the work directory on the `share` volume at `/share`? `share` volume has persistence.

Reply:
It does not really matter at the moment, as these temporary files are templates/written every time the role runs. One could even implement this without writing this to a file but I guess it is just nice for debugging purposes.